src/js/components/covid19/spendingOverTime/AwardSpendingOverTime.jsx

```diff
@@ -3,26 +3,55 @@
  * Created by Lizzie Salita 6/9/20
  */
 
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
+import { fetchNewAwardsCount } from 'helpers/disasterHelper';
 import SpendingOverTimeContainer from 'containers/covid19/spendingOverTime/SpendingOverTimeContainer';
 import AmountTab from './AmountTab';
 
 const tabs = [
     {
         type: 'obligations',
         label: 'Award Obligations',
-        description: 'How much was promised to be spent on COVID-19 response awards?'
+        description: 'How much was promised to be spent on COVID-19 response awards?',
+        dollarAmount: true
     },
     {
         type: 'outlays',
         label: 'Award Outlays',
-        description: 'How much was paid out to COVID-19 response awardees?'
+        description: 'How much was paid out to COVID-19 response awardees?',
+        dollarAmount: true
+    },
+    {
+        type: 'newAwards',
+        label: 'New Awards',
+        description: 'How many new awards were made in support of COVID-19 response?'
     }
 ];
 
 const AwardSpendingOverTime = () => {
     const [activeTab, setActiveTab] = useState('obligations');
-    // TODO - get total Award Obligations and Award Outlays from Redux when overview data is added
+    const [newAwards, setNewAwards] = useState(0);
+    const params = {
+        spending_type: 'award',
+        // TODO: remove hard-coded values after integration with v2/references/def_codes/ API
+        def_codes: ['L', 'M', 'N', 'O', 'P'],
+        fiscal_year: 2020
```

Comment:
Hard-coding the year? That seems dangerous...

Reply:
With all the iteration on this page it should be fine for MVP.

Reply:
FY filter has been removed